joomla/joomla-cms
# [5.1] Convert mod_toolbar to service provider

### Summary of Changes

Converts the toolbar administrator module to service provider.

### Testing Instructions

1. Make sure you have administrator module mod_toolbar published
2. You should see module output

![mod_toolbar](https://github.com/joomla/joomla-cms/assets/59166858/1285ca15-ebb9-45c9-aea9-b16bc70b2865)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
